cmd/config.go

```diff
@@ -526,57 +527,38 @@ func getAccountByName(name string) *account {
 	return nil
 }
 
-func getSelectedZone(number string, zones map[string]string) (string, bool) {
-	zName, ok := zones[number]
-	if !ok {
-		return "", false
-	}
-	return zName, true
-}
-
 func chooseZone(accountName string, cs *egoscale.Client) (string, error) {
-
-	reader := bufio.NewReader(os.Stdin)
-
 	zonesResp, err := cs.ListWithContext(gContext, &egoscale.Zone{})
 	if err != nil {
 		return "", err
 	}
 
-	zones := map[string]string{}
-
 	if len(zonesResp) == 0 {
 		return "", fmt.Errorf("no zones were found")
 	}
 
-	fmt.Printf("Choose %q default zone:\n", accountName)
+	zones := make([]string, len(zonesResp))
 
 	for i, z := range zonesResp {
 		zone := z.(*egoscale.Zone)
-
 		zName := strings.ToLower(zone.Name)
+		zones[i] = zName
+	}
 
-		n := fmt.Sprintf("%d", i+1)
-
-		zones[n] = zName
-
-		fmt.Printf("%d: %s\n", i+1, zName)
+	prompt := promptui.Select{
+		Label: fmt.Sprintf("Choose %q default zone", accountName),
```

Comment:
How about "Choose the default zone for %q" instead?

Reply:
👍